traffic_ops/traffic_ops_golang/routing/routes.go

```diff
@@ -355,6 +356,11 @@ func Routes(d ServerData) ([]Route, []RawRoute, http.Handler, error) {
 		{api.Version{4, 0}, http.MethodPost, `coordinates/?$`, api.CreateHandler(&coordinate.TOCoordinate{}), auth.PrivLevelOperations, Authenticated, nil, 44281121573},
 		{api.Version{4, 0}, http.MethodDelete, `coordinates/?$`, api.DeleteHandler(&coordinate.TOCoordinate{}), auth.PrivLevelOperations, Authenticated, nil, 43038498893},
 
+		//CDN notification
+		{api.Version{4, 0}, http.MethodGet, `cdn_notifications/?$`, cdnnotification.Read, auth.PrivLevelReadOnly, Authenticated, nil, 2221224514},
+		{api.Version{4, 0}, http.MethodPost, `cdn_notifications`, cdnnotification.Create, auth.PrivLevelOperations, Authenticated, nil, 2765223513},
```

Comment:
This one can't end with a `/`? why not? Should be anchored with a `$` either way - we've had that introduce weird bugs in the past when that wasn't done.

Reply:
what's the problem? this is how all GETs look

Reply:
what _should_ it look like?

Reply:
oh sorry, maybe i'm looking at the wrong one

Reply:
fixed. will push shortly.